sdk/test-utils/multi-version/README.md

```diff
@@ -0,0 +1,256 @@
+- Service API versions supported by the SDK library under test is specified for a top-level Mocha
+  `describe()` test suite. We loop through them and run the test suite on each version.
+  - Testing of multi-service-version support is only done in `live` test mode. For other test modes
+    (`record`, `soft-record`, and `playback`), tests are only running for the latest version, or a
+    chosen version specified via options in case the latest version is not the latest stable version.
+- Optionally a list or range of versions can be specified for a nested `describe()` test suite or
+  `it()` test case. If the version being tested is not in the range of versions supported by this
+  test suite/case then the test suite/case is skipped.
+
+## Get started
+
+We're about to go through how to set up your project to use the `@azure/test-utils-multi-version`
+package.
+
+This document assumes familiarity with [git](https://git-scm.com) and [rush](https://rushjs.io).
+You can read more about how we use rush in the following links:
+
+- Rush used for [Project Orchestration](https://github.com/sadasant/azure-sdk-for-js/blob/master/CONTRIBUTING.md#project-orchestration).
+- [Rush for NPM users](https://github.com/sadasant/azure-sdk-for-js/blob/master/CONTRIBUTING.md#rush-for-npm-users).
+
+Keep in mind that `@azure/test-utils-multi-version` is not a published package. It is only intended
+to be used by the libraries in the `azure-sdk-for-js` repository.
+
+### Installing the package
+
+To install the `@azure/test-utils-multi-version` package, you'll need to start by cloning our
+`azure-sdk-for-js` repository. One way of doing this is by using the git command line interface, as
+follows:
+
+```bash
+cd /path/to/my/github/repositories
+git clone https://github.com/Azure/azure-sdk-for-js/
+```
+
+Having cloned this repository, let's set it up by running the following rush commands:
+
+```bash
+cd azure-sdk-for-js
+rush update
+rush install
+rush build
+```
+
+This will optimistically assume you're in a fresh clone.
+
+From this point forward, we'll assume that you're developing (perhaps contributing!) to one of the
+`azure-sdk-for-js`'s libraries. So, your next step is to change directory to the path relevant to
+your project. Let's say you want to add the `@azure/test-utils-multi-version` package to
+`@azure/keyvault-keys`, you'll be doing the following:
+
+```bash
+cd sdk/keyvault/keyvault-keys
+```
+
+Once there, you can add the `test-utils-multi-version` package by changing your package.json
+to include the following line in the `devDependencies` section:
+
+```bash
+{
+  // ... your package.json properties
+  "devDependencies": {
+    // ... your devDependencies
+    "@azure/test-utils-multi-version": "^1.0.0",
+    // ... more of your devDependencies
+  },
+  // ... more of your package.json properties
+}
+```
+
+After that, we recommend you to update rush and install the dependencies again, as follows:
+
+```bash
+rush update && rush install
```

Comment:
`rush install` may not be needed?

Reply:
Sure, again copied from test-utils-recorder.

Reply:
removed